source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -216,12 +216,19 @@ class StopAreaSerializer(serpy.DictSerializer):
     timezone = NestedPropertyField(attr='properties.geocoding.timezone')
```

Comment:
It would be nice to add a test with a bragi mock missing  `geocoding` field, check that it doesn't crash on trying to get timezone attribute of NoneType ;)

Reply:
In the mock BRAGI_MOCK_STOP_AREA_WITH_BASIC_ATTRIBUTS attributs like timezone, commercial_modes, physical_modes, codes are absent

Reply:
I meant the whole `geocoding` itself.
But it should probably be delt with in another PR.